src/consts.ts

```diff
@@ -65,8 +73,16 @@ const SquareErrors = {
 };
 
 interface SocialMediaLink {
-  platform: 'facebook' | 'instagram' | 'wechat';
+  platform:
```

Comment:
at this point I'm fine just typing this a string -- or creating a separate enum if we really want to keep locks on this

Reply:
Replaced consts with imports and replaced list of platforms with string type since they're defined in the array later anyway.